provisioning/roles/appserver/tasks/main.yml

```diff
@@ -1,5 +1,9 @@
+- name: "Check if nodejs-legacy is needed"
+  set_fact:
+      nodejs_legacy_pkg: ',nodejs-legacy'
+  when: ( ansible_distribution == 'Ubuntu' and ansible_distribution_version is version('16.04', '<=')) or (ansible_distribution == 'Debian' and ansible_distribution_version is version('9', '<='))
 - name: Install uwsgi and python and other dependencies
-  apt: pkg=uwsgi,uwsgi-plugin-python3,python3-dev,virtualenv,python-pip,libpq-dev,nodejs-legacy,npm,git,gettext,libjpeg-dev state=installed update_cache=true
+  apt: pkg=uwsgi,uwsgi-plugin-python3,python3-dev,virtualenv,python-pip,libpq-dev,nodejs {{ nodejs_legacy_pkg }},npm,git,gettext,libjpeg-dev state=installed update_cache=true
```

Comment:
Two things:
* there's a space before ` {{ nodejs_legacy_pkg }}` that should probably be removed
* do we need `nodejs` when we're installing `nodejs-legacy`? Shouldn't it be one or the other?